pkg/server/event.go

```diff
@@ -0,0 +1,47 @@
+package server
+
+import (
+	"net/http"
+
+	"github.com/CovidShield/server/pkg/config"
+	"github.com/CovidShield/server/pkg/persistence"
+
+	"github.com/Shopify/goose/srvutil"
+	"github.com/gorilla/mux"
+)
+
+func NewEventServlet(cache persistence.RedisConn) srvutil.Servlet {
+	return &eventServlet{cache: cache}
+}
+
+type eventServlet struct {
+	cache persistence.RedisConn
+}
+
+func (s *eventServlet) RegisterRouting(r *mux.Router) {
+	r.HandleFunc("/event/nonce", s.eventNonce)
+}
+
+func (s *eventServlet) eventNonce(w http.ResponseWriter, r *http.Request) {
```

Comment:
Do we need to handle OPTIONS for this endpoint?

Reply:
Depends on if the browser does the call or something else in the OS.